sdk/events.go

```diff
@@ -19,6 +19,7 @@ type PushEvent struct {
 	Repository    PushEventRepository
 	AfterCommitID string `json:"after"`
 	Installation  PushEventInstallation
+	RepositoryURL string `json:"html_url"`
```

Comment:
Are you sure it's `html_url`? is there anything else? Just wondering about the html reference.

Reply:
I previously thought that the payload of `url` will be the ssh or https version depending on which URL you chose to push with but that was not the case. So I will swap with `url`.